packages/backend/src/services/HealthService/HealthService.mock.ts

```diff
@@ -31,70 +42,39 @@ export const BaseResponse = {
             loginPath: '',
         },
     },
-    defaultProject: undefined,
-    latest: { version: '0.2.7' },
-    hasEmailClient: false,
-    siteUrl: undefined,
-    intercom: undefined,
-    posthog: undefined,
-    rudder: undefined,
-    sentry: {
-        frontend: {
-            dsn: 'frontend-dsn.sentry.io',
-        },
-        environment: 'environment',
-        release: '1234',
+    intercom: {
+        apiBase: '',
+        appId: '',
     },
-    hasSlack: false,
-    hasHeadlessBrowser: false,
-    query: undefined,
-    staticIp: undefined,
-    hasDbtSemanticLayer: false,
-    hasGroups: false,
-    hasExtendedUsageAnalytics: false,
-    hasGithub: false,
-};
-
-export const Config = {
-    mode: LightdashMode.DEFAULT,
-    auth: {
-        disablePasswordAuthentication: false,
-        google: {
-            loginPath: '',
-            oauth2ClientId: '',
-            oauth2ClientSecret: '',
-            callbackPath: '',
-        },
-        okta: {
-            loginPath: '',
-        },
-        oneLogin: {
-            loginPath: '',
-        },
-        azuread: {
-            loginPath: '',
-        },
-        oidc: {
-            loginPath: '',
-        },
+    latest: {
+        version: '0.2.7',
     },
-    groups: {
-        enabled: false,
+    pivotTable: {
+        maxColumnLimit: 0,
     },
-    extendedUsageAnalytics: {
-        enabled: false,
+    posthog: {
+        apiHost: '',
+        projectApiKey: '',
+    },
+    pylon: {
+        appId: '',
+    },
+    query: {
+        csvCellsLimit: 100000,
+        maxLimit: 5000,
+    },
+    rudder: {
+        dataPlaneUrl: '',
+        writeKey: '',
     },
     sentry: {
-        backend: {
-            dsn: 'backend-dsn.sentry.io',
-        },
+        environment: '',
         frontend: {
-            dsn: 'frontend-dsn.sentry.io',
+            dsn: '',
         },
-        environment: 'environment',
-        release: '1234',
+        release: '',
     },
-} as LightdashConfig; // TODO: Refactor this to be a mock of the actual configuration, rather than a partial that might contain incorrect properties.
```

Comment:
Removed this mock and used the one from `import { lightdashConfigMock } from '../../config/lightdashConfig.mock';`. 

Update the `BaseResponse` mock to have proper typing and valid values.